css/event-detail.css

```diff
@@ -0,0 +1,91 @@
+/* everything is wonky - on my screen it cuts off on the left - on Gurjeets screen it cuts off on the right or works fine. Looks fine at size 390 */
+html {
+	box-sizing: border-box;
+}
+
+.template-wrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	/* justify-content: center; */
+	width: var(--mobile-wrapper-width);
+	margin: 0 auto;
+	max-width: var(--wrapper-width-max);
+}
+
+.template-wrapper img {
+	align-items: center;
+}
+
+.template-main-header {
+	text-align: center;
+	color: var(--color-onyx);
+	padding: 0 20px 0px 20px;
```

Comment:
There are several places in this file where we can replace the values with variables.

Reply:
done